static/src/javascripts-legacy/projects/common/modules/article/open-module.js

```diff
@@ -34,14 +34,14 @@ define([
         init: function () {
             if (config.page.openModule) {
                 spaceFiller.fillSpace(getSpacefinderRules(), function (spaces) {
-                    ajax({
-                        url: config.page.openModule,
-                        crossOrigin: true,
-                        method: 'get'
+                    fetch(config.page.openModule, {
+                        mode: 'cors',
                     }).then(function (resp) {
-                        if (resp.html) {
+                        var body = resp && resp.text();
+
+                        if (body) {
                             fastdom.write(function () {
-                                $.create(resp.html)
+                                $.create(body)
```

Comment:
Are `body` and `resp.html` equivalent? I wonder if this should be `$.create(body.html)`?